services/simcore/docker-compose.deploy.yml

```diff
@@ -14,6 +14,7 @@ services:
         - traefik.http.routers.webserver.middlewares=gzip_compress@file
     networks:
       - public
+      - monitored-net
```

Comment:
let us try to be consistent with names from the beginning. Either we add ``-net`` to all or not. I kind of like the minimalist approach of not but I leave the choice to you :-)

Reply:
renamed